cluster.go

```diff
@@ -86,7 +86,7 @@ func NewCluster(
 		return nil, err
 	}
 
-	logger.Infof("IPFS Cluster v%s listening on:", Version)
+	logger.Infof("IPFS Cluster v%s-%s listening on:", Version, Commit[0:8])
```

Comment:
As TestClusterStateSync doesn't hit the cluster service version command this slice dereference seems to be the cause of the slice out of bounds error that Travis catches for these changes.

Reply:
Thanks